src/components/views/messages/MLocationBody.tsx

```diff
@@ -93,16 +95,19 @@ export default class MLocationBody extends React.Component<IBodyProps, IState> {
             bodyId={this.getBodyId()}
             markerId={this.getMarkerId()}
             error={this.state.error}
+            tooltip={_t("Expand map")}
             onClick={this.onClick}
         />;
     }
 }
 
+
 interface ILocationBodyContentProps {
     mxEvent: MatrixEvent;
     bodyId: string;
     markerId: string;
     error: Error;
+    tooltip?: String;
```

Comment:
```suggestion
    tooltip?: string;
```

Reply:
Primitive vs boxed String instance

Reply:
Sorry, was writing Rust this morning :-)